test/pytest_runner.py

```diff
@@ -23,7 +23,7 @@ def test_module(name, env_module):
     if "classic/" not in name:
         max_cycles_test(env_module)
 
-    if "butterfly/" in name:
+    if ("butterfly/" or "mpe/") in name:
```

Comment:
Python does not work like this. It will evaluate `bool("butterfly/" or "mpe/")` -> `True`
and then ask if `True in name`.